docs/admin_api/room_membership.md

```diff
@@ -0,0 +1,34 @@
+# Edit Room Membership API
+
+The API allow an administrator to join an user account with a specific `user_id`
+to a room with a specific `roomIdOrAlias`.
+You can only modify local users.
+The room must have join rule `JoinRules.PUBLIC`. It is default for public rooms.
+The administrator must be admin or creator of a room, if the room has join rule
+`JoinRules.INVITE` (default for private rooms).
```

Comment:
Some minor word changes.

```suggestion
The API allows an administrator to join an user account with a given `user_id`
to a room with a given `roomIdOrAlias`. You can only modify the membership of
local users. The room must have join rule `JoinRules.PUBLIC`, which is the              
default for public rooms. If the room has the join rule `JoinRules.INVITE`
(default for private rooms), the server administrator must be a room admin or
the room creator.
```

Must the user be a room administrator, or just high enough power level to send invites (mods by default in private rooms)?

Reply:
Power level to send invites is high enough (member per default). I have added a test.